src/index.html

```diff
@@ -11,7 +11,234 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <!-- <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <div class="personal">
+          <label>
+            Surname:
+            <input
+              class="form-surname"
+              type="text"
+              name="surname"
+            >
+          </label>
+        </div>
+        <div class="personal">
+          <label>
+            Name:
+            <input
+              class="form-name"
+              type="text"
+              name="name"
+            >
+          </label>
+        </div>
+
+
+      </fieldset>
+    </form> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
```

Comment:
```suggestion
          <label>
            Surname:
            <input
              type="text"
              name="surname"
              required
              autocomplete="off"
            >
          </label>
```

`div` wrapper is redundant, you can leave only `label`